docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+| select_columns      | list[string]          | list of columns to be considered for the reconciliation process                                                                                                                                                                  | optional(default=None) | ["id","name","address"]                                                                                                        |
+| drop_columns        | list[string]          | list of columns to be eliminated from the reconciliation process                                                                                                                                                                 | optional(default=None) | ["comment"]                                                                                                                    |
+| column_mapping      | list[ColumnMapping]   | list of column_mapping,that helps in resolving column name mismatch b/w src and tgt eg: "id" in src  and "emp_id" in tgt.For more info [column_mapping](#column_mapping)                                                         | optional(default=None) | "column_mapping": [{"source_name": "id","target_name": "emp_id"}]                                                              |
+| transformations     | list[Transformations] | list of user-defined transformations ,that can be applied on src and tgt columns in case of any incompatibility data types or explicit transformation applied during migration.For more info [transformations](#transformations) | optional(default=None) | "transformations": [{"column_name": "s_address","source": "trim(s_address)","target": "trim(s_address)"}]                      |
+| thresholds          | list[Thresholds]      | list of threshold conditions, that can be applied on tgt columns to match the minor exceptions in data.It supports percentile,absolute, and date fields.For more info [thresholds](#thresholds)                                  | optional(default=None) | "thresholds": [{"column_name": "sal","lower_bound": "-5%","upper_bound": "5%","type": "integer"}]                              |
+| filters             | Filters               | filter expr that can be used to filter the data on src and tgt based on respective expressions                                                                                                                                   | optional(default=None) | "filters" : {"source":"lower(dept_name)>'it'" ,"target" : "lower(department_name)>'it'"}                                       |
```

Comment:
We have to mention saying that this filters should be in source dialect SQL format, and we don't manipulate these at all. we just blindly use them on objects. Either do it here or below

Reply:
added in Key Considerations on Table Config